server/itx/server.go

```diff
@@ -174,6 +174,11 @@ func StartServer(ctx context.Context, svr *Server, probeSvr *probe.Server, cfg c
 		log.L().Fatal("Failed to start server.", zap.Error(err))
 		return
 	}
+	defer func() {
+		if err := svr.Stop(ctx); err != nil {
+			log.L().Panic("Failed to stop server.", zap.Error(err))
+		}
+	}()
```

Comment:
stop() should be called in the LIFO order of start(), so use defer to do that

Reply:
Maybe change task to not to use defer to make it more clear? Just stop them in order at end of the function

Reply:
i think this way is also clear?

```
a.Start()
defer a.Stop()

b.Start()
defer b.Stop()

c.Start()
defer c.Stop()

<- channel to receive exit or cancel signal
```
then we don't need to worry about the exact order. defer is exactly designed to handle this kind of usage.